packages/common/src/HardhatConfig.js

```diff
@@ -67,6 +73,28 @@ function getHardhatConfig(configOverrides, workingDir = "./") {
         url: getNodeUrl("mainnet", true),
         accounts: { mnemonic },
       },
+      optimism: {
+        url: "http://127.0.0.1:8545",
```

Comment:
`optimism` needs to run against a local node. The best way to run a node currently is to pull a docker container. Instructions [here](https://github.com/ethereum-optimism/optimism#development-quick-start).

We'll need to think about how to integrate this into our CI test. Note that building the docker container took me ~10 minutes so we need to think of a different solution than building it on each run.